### PR TITLE
chore: update Server and Custom Stickers

### PR DESCRIPTION
**Pull request by NMINHDAI**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

**Review by trkhoi — APPROVED** [review]

File: migrations/schemas/20220505151937-create_discord_stats.sql [hunk 1]
```diff
@@ -20,8 +20,8 @@ CREATE TABLE "discord_guild_stats" (
     "nr_of_animated_emojis" INTEGER,
 
     "nr_of_stickers" INTEGER,
-    "nr_of_standard_stickers" INTEGER,
-    "nr_of_guild_stickers" INTEGER,
+    "nr_of_custom_stickers" INTEGER,
```
Comment (trkhoi) [inline migrations/schemas/20220505151937-create_discord_stats.sql:23]: create new migration

File: migrations/schemas/20220527140133-change-column-name-stickers.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+-- +migrate Up
+ALTER TABLE "discord_guild_stats"
+    RENAME COLUMN nr_of_guild_stickers TO nr_of_server_stickers;
+    
+ALTER TABLE "discord_guild_stats"
```
Comment (trkhoi) [inline migrations/schemas/20220527140133-change-column-name-stickers.sql:6]: move this to `mograte down`

File: migrations/schemas/20220527140133-change-column-name-stickers.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    RENAME COLUMN nr_of_standard_stickers TO nr_of_custom_stickers;
+
+-- +migrate Down
+
```
Comment (trkhoi) [inline migrations/schemas/20220527140133-change-column-name-stickers.sql:10]: migrate down to add back old fields when rollback, ex:
- Migrate up: `ALTER TABLE "discord_guild_stats"
    RENAME COLUMN nr_of_guild_stickers TO nr_of_server_stickers;`
- Migrate down: `ALTER TABLE "discord_guild_stats"
    RENAME COLUMN nr_of_server_stickers TO nr_of_guild_stickers;`
